src/Xamarin.Android.Build.Tasks/packages.config

```diff
@@ -3,4 +3,17 @@
   <package id="FSharp.Compiler.CodeDom" version="1.0.0.1" targetFramework="net45" />
   <package id="FSharp.Core" version="3.1.2.5" targetFramework="net451" />
   <package id="Irony" version="0.9.1" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="11.0.1" targetFramework="net451" />
+  <package id="NuGet.Common" version="4.6.0" targetFramework="net462" />
```

Comment:
You need to switch to PackageReference, this would have been two entries or so, with no changes in the csproj ;)